server/src/routes/api/user.mjs

```diff
@@ -84,9 +85,76 @@ function getUsers(req, res) {
   });
 }
 
+// POST /api/v1/state/enableInteractionLogs
```

Comment:
This does not seem like it belongs in user.mjs, which is supposed to be functionality related to per-user functionality/state. Especially given the route, I think this probably belongs in state.mjs.

Reply:
done